src/Rule/Boolean.php

```diff
@@ -39,9 +39,6 @@ class Boolean extends Rule
      */
     public function validate($value)
     {
-        if ($value === true || $value === false) {
-            return true;
-        }
-        return $this->error(self::NOT_BOOL);
+        return is_bool($value) ?: $this->error(self::NOT_BOOL);
```

Comment:
From readability point of view while comparing `is_bool($value)` VS `$value === true || $value === false` IMO first variation win. Also i do not think its good idea to avoid use typical functions from any point of view.